templates/todo/api/java/src/main/java/com/microsoft/azure/simpletodo/configuration/WebConfiguration.java

```diff
@@ -22,27 +29,42 @@ public WebMvcConfigurer webConfigurer() {
         return new WebMvcConfigurer() {
             @Override
             public void addCorsMappings(CorsRegistry registry) {
-                String apiUrl = System.getenv("REACT_APP_WEB_BASE_URL");
-
-                if (apiUrl != null) {
-                    String localHostUrl = "http://localhost:3000/";
-                    registry.addMapping("/**").allowedOrigins("https://portal.azure.com",
-                            "https://ms.portal.azure.com", localHostUrl,
-                            apiUrl).allowedMethods("*").allowedHeaders("*");
-                    String fileName = Thread.currentThread().getStackTrace()[1].getFileName();
-                    File file = new File(fileName);
-                    String absolutePath = file.getAbsolutePath();
-                    System.out.println(
-                            "CORS with " + localHostUrl
-                                    + " is allowed for local host debugging. If you want to change pin number, go to "
-                                    + absolutePath);
+                // env.ENABLE_ORYX_BUILD is only set on Azure environment during azd provision for todo-templates
+                // You can update this to env.JAVA_ENV if your app is using `development` to run locally and another value
+                // when the app is running on Azure (like production or stating)
+                String runningOnAzure = System.getenv("ENABLE_ORYX_BUILD");
```

Comment:
We default orxy build based on platform `param enableOryxBuild bool = contains(kind, 'linux')`. The motivation here escapes me. Does this change how the implementation of Azure detection?

Reply:
We don't really care about what value is set for `ENABLE_ORYX_BUILD`, we are only using it as a sentinel to know if the service is running on Azure or not...

What I am thinking now is if this won't work for Container Apps...  this env var won't be found and the app will think it's running locally, and there won't be any cors config :(